src/lib.rs

```diff
@@ -151,16 +151,18 @@ extern crate derive_more;
 
 pub mod bench_utils;
 pub(crate) mod circuit;
+pub mod config;
 pub mod constants;
 pub mod errors;
 pub mod freeze;
 pub mod keys;
 pub mod mint;
 pub mod parameters;
+pub mod prelude;
 pub mod proof;
 pub mod structs;
 pub mod transfer;
-mod types;
+// mod types;
```

Comment:
Also was there a file called `types.rs` before?

Reply:
because it contains instantiation of all types regarding curve choice and structs that are generic over curve choices.

It's supposed to make our code non-generic and simple-looking (without the generic parameter everywhere)

Reply:
Oh I meant that I didn't find this file in the PR (i.e. the code before changing). Is this file supposed to be deleted?

Reply:
yes, it's supposed to be deleted, which I will do after I update all other places. I'm temporarily keeping the file for my reference, but commenting it out so that all previous code that depend on this file will throw me error and I know where to fix.